data/json/npcs/robofac/NPC_Cranberry_Foster.json

```diff
@@ -0,0 +1,803 @@
+      { "trait": "hair_blond_medium" },
+      { "trait": "eye_blue" },
+      { "trait": "SKIN_PINK" }
+    ],
+    "bonus_str": 2,
+    "bonus_dex": 3,
+    "bonus_per": 1,
+    "worn_override": "NC_ROBOFAC_MERC_1_worn",
+    "carry_override": "NC_ROBOFAC_MERC_1_carry",
+    "weapon_override": "NC_ROBOFAC_MERC_1_wield",
+    "skills": [
+      { "skill": "gun", "bonus": 4 },
+      { "skill": "rifle", "bonus": 4 },
+      { "skill": "pistol", "bonus": 1 },
+      { "skill": "melee", "bonus": 4 },
+      { "skill": "cutting", "bonus": 3 },
+      { "skill": "fabrication", "bonus": 1 },
+      { "skill": "firstaid", "bonus": 1 },
+      { "skill": "swimming", "bonus": 3 },
+      { "skill": "survival", "bonus": 4 },
+      { "skill": "cooking", "bonus": 4 },
+      { "skill": "computer", "bonus": 6 },
+      { "skill": "electronics", "bonus": 4 },
+      { "skill": "driving", "bonus": 3 }
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "NC_ROBOFAC_MERC_1_worn",
+    "subtype": "collection",
+    "items": [
+      { "item": "under_armor" },
+      { "item": "under_armor_shorts" },
+      { "item": "ring_wedding" },
+      { "item": "robofac_enviro_suit" },
+      { "item": "slingpack" },
+      { "item": "robofac_kevlar_vambraces" },
+      { "item": "robofac_kevlar_greaves" },
+      { "item": "knife_combat", "container-item": "bootsheath" },
+      { "group": "robofac_mantle_merc" }
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "NC_ROBOFAC_MERC_1_carry",
+    "subtype": "collection",
+    "items": [
+      { "item": "ear_plugs", "custom-flags": [ "no_auto_equip" ] },
+      { "item": "robofac40", "ammo-item": "556", "charges": 40 },
+      { "item": "robofac_gun_kit" },
+      { "item": "grenade_emp" },
+      { "item": "bot_grenade_hack", "count": 2 },
+      { "item": "protein_bar_evac", "count": 7 },
+      { "item": "water_clean", "count": 2 },
+      { "item": "RobofacCoin", "count": [ 15, 18 ] },
+      { "group": "charged_smart_phone" },
+      { "group": "full_1st_aid" }
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "NC_ROBOFAC_MERC_1_wield",
+    "subtype": "collection",
+    "items": [
+      { "item": "robofac_gun", "ammo-item": "556", "charges": 40, "contents-item": [ "shoulder_strap", "robofac_gun_ar" ] }
+    ]
+  },
+  {
+    "id": "TALK_ROBOFAC_MERC_1",
+    "type": "talk_topic",
+    "dynamic_line": {
+      "u_has_var": "u_met_Cranberry",
+      "type": "general",
+      "context": "meeting",
+      "value": "yes",
+      "no": "&There's a person reclined on the sofa here.  They're very tall, with their legs reaching to the other armrest, and they're wearing a set of Hub 01's modular armor.  An environmental suit in blue-and-brown is crumpled in a misshapen heap on the floor near the table, and leaning against the far armrest is a rifle looking like nothing you've seen before.\n\nThey look up towards you as you come through the door and wave a hand.  \"Placeholder text.\"",
```

Comment:
ah yes, the most common greeting after the cataclysm: "placeholder text".

Reply:
It's deliberate - see the dialogue that directly has the player responding to it. :p